cargo-dist/src/build/cargo.rs

```diff
@@ -25,7 +31,23 @@ impl<'a> DistGraphBuilder<'a> {
             .workspace(workspace_idx)
             .workspace_dir
             .clone();
+
         for (binary_idx, binary) in self.inner.binaries.iter().enumerate() {
+            let package = self.workspaces.package(binary.pkg_idx);
+
+            if let Some(axoproject::Version::Cargo(version)) = &package.axoupdater_version {
+                let axoupdater_min_version = semver::Version::parse(AXOUPDATER_MINIMUM_VERSION)
+                    .expect("invalid axoupdater const?!");
+
+                if *version < axoupdater_min_version {
+                    return Err(DistError::AxoupdaterTooOld {
```

Comment:
if we error on transitive axoupdater deps, we should make sure we tell them how it is in their dependency tree

Reply:
Updated:

```
  × Your project (cargo-dist) uses axoupdater as a library, but the version specified (0.7.2) is older than the
  │ minimum supported version (0.10.0). (The dependency comes via cargo-dist in the dependency tree.)
  help: https://opensource.axo.dev/cargo-dist/book/installers/updater.html#minimum-supported-updater-version-
        checking
```